autogluon/utils/tabular/ml/trainer/abstract_trainer.py

```diff
@@ -294,14 +297,14 @@ def train_and_save(self, X_train, y_train, X_val, y_val, model: AbstractModel, s
             logger.log(20, err)
             del model
         else:
-            self.add_model(model=model, stack_name=stack_name, level=level)
+            self._add_model(model=model, stack_name=stack_name, level=level)
             model_names_trained.append(model.name)
             if self.low_memory:
                 del model
         return model_names_trained
 
-    def add_model(self, model: AbstractModel, stack_name: str, level: int):
-        stack_loc = self.models_level[stack_name]  # TODO: Consider removing, have train_multi handle this
+    def _add_model(self, model: AbstractModel, stack_name: str, level: int):
```

Comment:
add documentation about role of `stack_name`, `level` args

Reply:
added